examples/list-devices.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+# Import project path
+import sys
+import os
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
```

Comment:
Great to include some examples - but why do you need this?

Reply:
There’s probably a better way that I’m just not aware of, but this is an attempt to import the project path in a way that is portable so that you can run the script from anywhere and it will find the module appropriately relative to the example file.